client/package.json

```diff
@@ -20,7 +20,7 @@
   "eslintConfig": {
     "extends": "react-app"
   },
-  "proxy": "http://localhost:3001",
+  "proxy": "http://localhost:5000",
```

Comment:
yeah let's turn all these back and fix the structure of the app to the original way